src/serialize.jl

```diff
@@ -0,0 +1,27 @@
+using ..ChemistryFeaturization.AbstractType: AbstractAtoms, AbstractFeaturization
+
+"""
+    SerializableEncodedFeature
+
+Helps in serialization, and keeps a record of the `featurization` applied to an `atoms`, and the `encoded_features`
```

Comment:
Can we change this to something more general too, like

> Container object for an `Atoms` object, a featurization, and the resulting encoded features from applying the featurization to the atoms.

Reply:
Yes, that makes more sense to me.

Reply:
Resolved in 18893cf9dad7d546f3eff93a005abe212957440d.